build.xml

```diff
@@ -24,145 +24,138 @@
 <project name="romraider" default="help" basedir=".">
 
     <!-- get the environment -->
-    <property environment="env"/>
+    <property environment="env" />
 
     <!-- =================================================================== -->
     <!-- Help                                                                -->
     <!--   All externally callable targets should be listed here, along      -->
     <!--   with common property overrides                                    -->
     <!-- =================================================================== -->
     <target name="help" depends="-init">
-        <echo message="help        [Print this message                     ]"/>
```

Comment:
the fact that you reformat the whole file makes it harder to see the actual patch. would it be safer to not reformat the whole file and only change the lines you want to change?

Reply:
I actually wanted to change all the lines I changed.  This is why I suggested turning off whitespace when diffing.  Then, you'll see just the actual content change.  The space before the close of the XML tags is more proper.

Reply:
I've learned something today. With "diff ignoring whitespace" looks much better, thank you!

Reply:
Glad to point out something new!  It's really useful in cases where you have a git pre-commit hook that adjusts indentation to a particular standard or strips whitespace at the end of a line, or times where someone is just cleaning up for consistency in a file.